z_infra_provisioning/cloud_infra_center/ocp_upi/inventory.yaml

```diff
@@ -41,8 +41,8 @@ all:
       disk_type: 'dasd' # dasd or scsi
       #volume_type_id: '<storage-template-id>'
       
-      openshift_version: '4.7'
-      openshift_minor_version: '7'
+      openshift_version: '4.11'
+      openshift_minor_version: '9' #latest
```

Comment:
We may set {openshift_minor_version} default value to latest.

Reply:
update it.